Engines/Wine/QuickScript/Quick Script/script.js

```diff
@@ -90,7 +90,11 @@ module.default = class QuickScript {
     }
 
     wineVersion(wineVersion) {
-        this._wineVersion = wineVersion;
+        if (wineVersion && wineVersion instanceof Function) {
+            this._wineVersionFunction = wineVersion;
+        } else {
+            this._wineVersionFunction = function () { return wineVersion; };
```

Comment:
`function() {return wineVersion; }` or `() => wineVersion` ?

Reply:
I think it's purely a question of personal preference. Let's go with the current implementation.